torch_np/random.py

```diff
@@ -0,0 +1,120 @@
+"""Wrapper to mimic (parts of) np.random API surface.
+
+NumPy has strict guarantees on reproducibility etc; here we don't give any.
+
+Q: default dtype is float64 in numpy
+
+"""
+from math import sqrt
+
+import torch
+
+from . import asarray
+from ._detail._scalar_types import default_float_type as _default_float_type
+from ._detail import _util
+
+_default_dtype = _default_float_type.torch_dtype
+
+__all__ = ["seed", "random_sample", "sample", "random", "rand", "randn", "normal"]
+
+
+def seed(seed=None):
+    if seed is not None:
+        torch.random.manual_seed()
+
+
+def random_sample(size=None):
```

Comment:
Why not simply calling this `random` or `sample`?

Reply:
Numpy has these three aliases (random, random_sample and sample), all three are here.

Reply:
ah, fair enough!